rancher/charts
# Set y axis format to 's' for disk sync duration graphs.

## Issue: https://github.com/rancher/rancher/issues/40602

## Problem
Y axis label for Disk Sync Duration graphs in `General/Rancher/etcd (nodes)` dashboard set to ms. 

## Solution
Set y axes label for Disk Sync Duration in `General/Rancher/etcd (nodes)` dashboard to s. 

## Testing
1. Install monitoring chart v103.0.0+up40.1.2
2. Check that Disk Sync Duration graphs y axes are set to s in Grafana.
